src/main/java/by/issoft/mandrik/steps/LoginSteps.java

```diff
@@ -0,0 +1,35 @@
+package by.issoft.mandrik.steps;
+
+import by.issoft.mandrik.driver.DriverSingleton;
+import org.openqa.selenium.WebDriver;
+
+import by.issoft.mandrik.pages.LoginPage;
+
+public class LoginSteps {
+    private WebDriver driver;
+
+    public void initBrowser() {
```

Comment:
Methods for WebDriver should be in another class named WebDriverSteps or something else.